test/adapters/sequelize.test.js

```diff
@@ -110,14 +114,76 @@ function getField(schema, name) {
             field: 'createdAt',
             type: 'Date',
             columnName: 'createdAt',
+            isGenerated: true,
           });
           expect(getField(schema, 'updatedAt')).toStrictEqual({
             field: 'updatedAt',
             type: 'Date',
             columnName: 'updatedAt',
+            isGenerated: true,
           });
         });
       });
     });
+
+    describe('with non-autogenerated ids', () => {
+      const model = sequelize.define('post', {
+        id: {
+          type: Sequelize.UUID,
+          primaryKey: true,
+        },
```

Comment:
What if `id` is an auto incremented field:
```
id: {
  type: DataTypes.INTEGER,
  primaryKey: true,
  autoIncrement: true,
},
```
Should it be required too?

Reply:
Nope, because it is generated. In this test, the value is not automatically generated, and is used as a primary key.

The corresponding test is in this file, [line 67](https://github.com/ForestAdmin/forest-express-sequelize/pull/482/files#diff-51e5c93e4c5bb3adb928fc6769cfa369R67)

```
          expect(getField(schema, 'id')).toStrictEqual({
            field: 'id',
            type: 'Number',
            columnName: 'id',
            primaryKey: true,
            isGenerated: true,
          });
```